### PR TITLE
Exclude possibility of empty routes.

### PR DESCRIPTION
**Pull request by siarheishut**
Make empty routes unacceptable and fix tests.

### CODE REVIEW

File: tests/map_renderer_test.cpp [hunk 1]
```diff
@@ -188,25 +183,22 @@ TEST(TestMapRenderer, TestGetMap) {
...
                       ROUTE_END()
+                      ROUTE("rgba(210,81,14,0.94)")
+                      POINT(242.1, 214.631)
+                      ROUTE_END()
+                      ROUTE("purple")
+                      POINT(218.01, 192.764)
+                      ROUTE_END()
+                      ROUTE("rgb(15,23,41)")
+                      POINT(170, 170)
+                      ROUTE_END()
```
Comment (foxfrikses) [inline tests/map_renderer_test.cpp:194]: Please extract these changes to another commit. This one is only about disabling empty routes

Comment (siarheishut) [reply tests/map_renderer_test.cpp]: Changed.

**Review by foxfrikses — APPROVED** [review]
LGTM
